test/scripts/openshift-ci/run-e2e-tests.sh

```
@@ -46,26 +64,27 @@ cp ./test/e2e/conftest.py ./test/e2e/conftest.py.bak
 trap 'mv ./test/e2e/conftest.py.bak ./test/e2e/conftest.py 2>/dev/null || true' EXIT
 
 : "${SETUP_E2E:=true}"
-
 if [ "$SETUP_E2E" = "true" ]; then
   echo "Installing on cluster"
   pushd $PROJECT_ROOT >/dev/null
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)